assets/js/main.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+var weatherAppModule = (function() {
+  const displayWeatherData = (data, unit) => {
+    const cityName = document.getElementById("citytitle");
+    const content1 = document.getElementById("content1");
+    const content2 = document.getElementById("content2");
+    const card = document.getElementsByClassName("card")[0];
+    let temperatureUnit;
+
+    unit === "metric"
+      ? (temperatureUnit = "Celcius")
+      : (temperatureUnit = "Fahrenheit");
+
+    const { name, main, weather } = data;
+
+    const text = `
+     The temperature is ${Math.floor(main.temp)} 
+     ${temperatureUnit} , a humidity of ${Math.floor(main.humidity)} % 
+     and a pressure of ${Math.floor(main.pressure)} hPa`;
+
+    const text2 = document.createElement("p");
+
+    cityName.innerText = name;
+    content1.innerText = text;
+    text2.innerText = weather[0].description;
+    content2.innerText = "";
+
+    const img = document.createElement("img");
+    img.setAttribute(
+      "src",
+      `http://openweathermap.org/img/w/${weather[0].icon}.png`
+    );
+
+    content2.append(text2, img);
+
+    if (card.style.display === "") {
+      card.style.display = "block";
```

Comment:
Let's keep all our styles in the CSS file and use javascript to add/remove class names on elements as needed.